### ISSUE TITLE
Gamom drop not in loot table.

### ISSUE THREAD

**Issue by tabakisp** [chromiecraft/chromiecraft#451]
#### WHAT CLIENT DO YOU PLAY ON?

- [ ] enGB
- [x] enUS
- [ ] other (specify)

##### FACTION

- [ ] Alliance
- [x] Horde

##### CONTENT PHASE:

- [x] Generic
- [ ] 1-19
- [ ] 20-29
- [ ] 30-39
- [ ] 40-49
- [ ] 50-59

##### CURRENT BEHAVIOUR:

[Gamon](https://wowgaming.altervista.org/aowow/?npc=6466) is a killable NPC located in [Orgrimmar](https://wowgaming.altervista.org/aowow/?zone=1637). Killing him dropped a [Thistlefur Jerkin](https://wowgaming.altervista.org/aowow/?item=14202) of the Owl.

##### EXPECTED BLIZZLIKE BEHAVIOUR:

[Thistlefur Jerkin](https://wowgaming.altervista.org/aowow/?item=14202) is not on the loot table of [Gamon](https://wowgaming.altervista.org/aowow/?npc=6466).

##### STEPS TO REPRODUCE THE PROBLEM:

1. Go to [Orgrimmar](https://wowgaming.altervista.org/aowow/?zone=1637)
2. Pray to RNG
3. Kill [Gamon](https://wowgaming.altervista.org/aowow/?npc=6466)

##### EXTRA NOTES:

I have killed him countless times. It's the only time he dropped a green item for me.

 ```
 Player selected NPC
 DB GUID: 12325, current GUID: 12325.
 Faction: 7.
 npcFlags: 0.
 Entry: 6466.
 DisplayID: 5205 (Native: 5205).
 Level: 12.
 EquipmentId: 0 (Original: 0).
 Health (base): 247. (max): 247. (current): 247.
 Unit Flags: 33280.
 Unit Flags 2: 2048.
 Dynamic Flags: 0.
 Faction Template: 7.
 SpawnTime: Full:5m Remain:0s
 Loot: 6466 Pickpocket: 6466 Skinning: 0
 InstanceID: 0
 Phasemask: 1
 Armor: 562
 Position: 1637.520020 -4438.049805 15.733437.
 AIName:  ScriptName: 
 MechanicImmuneMask: 0
```
![Screenshot_20210421_153925](https://user-images.githubusercontent.com/7551798/115556602-86412d80-a2b9-11eb-8d42-f875e3d6649d.png)

##### AC HASH/COMMIT:

https://github.com/chromiecraft/azerothcore-wotlk/commit/b8b5f68f98ff8771a6ec87ea6805ceced91c406a

##### OPERATING SYSTEM:

Ubuntu 20.04

##### MODULES:

- [mod-cfbg](https://github.com/azerothcore/mod-cfbg)
- [mod-duel-reset](https://github.com/azerothcore/mod-duel-reset)
- [mod-desertion-warnings](https://github.com/azerothcore/mod-desertion-warnings)
- [mod-ah-bot](https://github.com/azerothcore/mod-ah-bot)
- [mod-eluna-lua-engine](https://github.com/azerothcore/mod-eluna-lua-engine)
- [mod-server-auto-shutdown](https://github.com/azerothcore/mod-server-auto-shutdown)
- [lua-LevelUpReward](https://github.com/55Honey/Acore_LevelUpReward)
- [lua-CarbonCopy](https://github.com/55Honey/Acore_CarbonCopy)

##### OTHER CUSTOMIZATIONS:

None.

##### SERVER:

ChromieCraft

**Comment by mdawx** [comment]
I opened you first link:

![image](https://user-images.githubusercontent.com/81375770/115612363-77736e80-a2eb-11eb-86ba-200f13bbc091.png)

![image](https://user-images.githubusercontent.com/81375770/115612278-60348100-a2eb-11eb-938e-fe1a42b385a9.png)

To be honest i don't know what is reliable database. Maybe we should dig into wowhead archives.

**Comment by tabakisp** [comment]
@mdawx yes you are right. Even though I did search for it, maybe I did a spelling mistake. Thanks for helping.
